iree/compiler/Dialect/Modules/Strings/strings.imports.mlir

```diff
@@ -27,9 +27,18 @@ vm.module @strings {
 vm.import @i32_to_string(%value : i32) -> !vm.ref<!strings.string>
 attributes {nosideeffects}
 
+// Returns a tensor of strings of elementwise to string of
```

Comment:
Reword. Elementwise conversion of a tensor to a tensor of strings.

Reply:
Comment updated.